src/main/scala/edu/berkeley/ce/rockslicing/CommandReader.scala

```diff
@@ -0,0 +1,54 @@
+package edu.berkeley.ce.rockslicing
+
+/** A simple tool for parsing command line inputs
```

Comment:
Let's go ahead and put a link back to the StackOverflow post that we based this on, just so we can easily find it later and just to be good about citing our sources.